config/custom-environment-variables.ts

```diff
@@ -17,10 +17,12 @@ export default {
       "S3_DRIVER_TESTS": "S3_DRIVER_TESTS"
     }
   },
-  "GETH_NODE": "GETH_NODE",
+  "DATABASE_URL": "DATABASE_URL",
   "ELASTICSEARCH_URL": "ELASTICSEARCH_URL",
+  "ETHERSCAN_API_KEY": "ETHERSCAN_API_KEY",
+  "GETH_NODE": "GETH_NODE",
   "INFLUXDB_URL": "INFLUXDB_URL",
+  "INFURA_PROJECT_ID": "INFURA_PROJECT_ID",
```

Comment:
I just want to check - is this something we need to configure for our deployed environments? Right now we just use the project-specific links and put them in the GETH_NODE variable

Reply:
Yes, we should set these in our deployment configuration.